src/blockchain/rpc.rs

```diff
@@ -294,6 +295,38 @@ impl WalletSync for RpcBlockchain {
         }) {
             let txid = tx_result.info.txid;
             list_txs_ids.insert(txid);
+            if tx_result.detail.category == GetTransactionResultDetailCategory::Immature {
+                // Immature UTXOs don't appear in rpc's listunspent, so we
+                // have to manually add them to the current_utxos list
+                // FIXME: This will produce a wrong result if the immature utxo is spent,
+                // which *should* be impossible.
```

Comment:
I don't think this warning is necessary, it's impossible to spend immature stuff